agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorMetricExporter.java

```diff
@@ -44,23 +53,53 @@
 import io.opentelemetry.sdk.metrics.data.PointData;
 import io.opentelemetry.sdk.metrics.export.AggregationTemporalitySelector;
 import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.io.File;
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Consumer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class AzureMonitorMetricExporter implements MetricExporter {
 
   private static final List<String> EXCLUDED_METRIC_NAMES = new ArrayList<>();
 
-  private final TelemetryClient telemetryClient;
+  private final String instrumentationKey;
+  private final TelemetryItemExporter telemetryItemExporter;
   private static final Logger logger = LoggerFactory.getLogger(AzureMonitorMetricExporter.class);
   private final AtomicBoolean stopped = new AtomicBoolean();
-
-  public AzureMonitorMetricExporter(TelemetryClient telemetryClient) {
-    this.telemetryClient = telemetryClient;
+  private final Consumer<AbstractTelemetryBuilder> defaultPopulator;
+
+  public AzureMonitorMetricExporter(
+      HttpPipeline httpPipeline,
+      URL endpoint,
+      String instrumentationKey,
+      Consumer<AbstractTelemetryBuilder> defaultPopulator) {
```

Comment:
I think it may be better to inject `TelemetryItemExporter` here, so that we can share that, instead of creating separate one here